dbt/include/athena/macros/materializations/models/table/create_table_as.sql

```diff
@@ -49,36 +49,31 @@
 
   {% do adapter.delete_from_s3(location) %}
 
-  create table {{ relation }}
-  with (
-    table_type='{{ table_type }}',
-    is_external={%- if table_type == 'iceberg' -%}false{%- else -%}true{%- endif %},
+  {# `create table table_name with (< properties >) as` should be on 1 line to help the athena engine v3 parser recognize UDF declarations. See Issue #274 #}
+  {# please be mindful of editing whitespace control characters #}
+  create table {{ relation }} with (table_type='{{ table_type }}', is_external={%- if table_type == 'iceberg' -%}false{%- else -%}true{%- endif -%}, format='{{ format }}'
```

Comment:
Instead of putting everything on one line, which will make it hard to maintain, what about something like this ?

```sql
{%- if not work_group_output_location_enforced or table_type == 'iceberg' -%}
    {{ location_property }}='{{ location }}',{{' '}}
  {%- endif -%}
```

Not sure it works 100% but it might worth a try